node/tests/SharedTests.ts

```diff
@@ -28,16 +36,6 @@ import {
     intoArray,
     intoString,
 } from "./TestUtilities";
-import { SingleNodeRoute } from "../build-ts/src/GlideClusterClient";
-import {
-    BitmapIndexType,
-    BitOffsetOptions,
-} from "../build-ts/src/commands/BitOffsetOptions";
-import { LPosOptions } from "../build-ts/src/commands/LPosOptions";
```

Comment:
You don't re-reexport option classes/types - don't we need them anymore? Does a user need in them?

Reply:
No because users don't use `new LPosOptions`, they create object with `{}`.